briefcase/django.py

```diff
@@ -78,10 +82,35 @@ def install_extras(self):
         # Install additional elements required for Django
         print(" * Installing extras...")
         print("   - Installing NPM requirements...")
-
-        npm = shutil.which("npm")
-        subprocess.Popen([npm, "install"], cwd=os.path.abspath(self.dir)).wait()
-
+        node_download_link = {
+            'darwin': 'https://nodejs.org/dist/v6.10.3/node-v6.10.3.pkg',
+            'linux' : '',
+            'win32' :'https://nodejs.org/dist/v6.10.3/node-v6.10.3-x86.msi',
+        }
+        system_platform = sys.platform
+        required_node_version = '6'
```

Comment:
There should also be a required_npm_version; the default version installed by node 6 is npm 3, which has problems.